common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -57,6 +57,30 @@
 ASSET_IGNORE_REGEX = getattr(settings, "ASSET_IGNORE_REGEX", r"(^\._.*$)|(^\.DS_Store$)|(^.*~$)")
 
 
+class MutableSignal(django.dispatch.Signal):
```

Comment:
lol, wow. I don't know how I missed that connotation. Will change.

Honestly, this whole PR is half baked. I'm just trying to see how big an effect it would have on the run.